config/cosmosdb/config.go

```diff
@@ -40,6 +45,14 @@ func Configure(p *config.Provider) {
 			},
 		}
 		r.UseAsync = true
```

Comment:
```suggestion
		r.UseAsync = true
		r.Kind = "SQLContainer"
```

And a find&replace will be needed to update `Type: "SqlContainer"` fields in this file.